src/commands/cmd_explain.c

```diff
@@ -45,12 +45,13 @@ int MGraph_Explain(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     pthread_setspecific(_tlsASTKey, ast);
 
     // Retrieve the GraphContext and acquire a read lock.
+    // RedisModule_ThreadSafeContextLock(ctx);
     gc = GraphContext_Retrieve(ctx, argv[1]);
+    // RedisModule_ThreadSafeContextUnlock(ctx);
```

Comment:
I don't think we require the lock here. I guess I don't have a preference on whether these commented lines should be deleted, but I'm leaving a comment so that you can choose whether to delete them or not!